core/performance_test/peakflops/03_Cabana_peakflops.cpp

```diff
@@ -13,16 +13,9 @@
 
 #include <cstdlib>
```

Comment:
How is this file related to what is in the new `performance_test` directory? I think `peakflops` should only appear in a single location.

Reply:
Seems reasonable. I don't really want to call it `Cabana.cpp` either :P. suggestions?

Reply:
Well I guess my point is why duplicate it? If we have peak flops as a performance test and it carries all the functionality of what is in `example/benchmark` then we should just delete what is in `example/benchmark`.

Reply:
Oh, my misunderstanding. Honestly, I thought I did:

https://github.com/rfbird/Cabana/tree/performance_tests/core/example/benchmark

Maybe I messed up the merge, so it's "not there" rather than "deleted"?

Reply:
Could be - one of the reasons I was curious how clean the merge/rebase was.